src/main/java/io/stargate/sgv2/jsonapi/exception/mappers/ThrowableToErrorMapper.java

```diff
@@ -46,6 +50,10 @@ public final class ThrowableToErrorMapper {
         }
 
         // handle all other exceptions
+        logger.error(
+            String.format(
+                "Unrecognized exception caught, mapped to SERVER_UNHANDLED_ERROR: %s", message),
+            throwable);
```

Comment:
Will log, including stack trace, when `Throwable` passed.

Reply:
possible for this to be tied to tenant ID?

Reply:
Will try to see: cannot pass directly (one of things I dislike about Dependency Injection style), but perhaps there's a way with context (MDC?)

Reply:
Looks like we do set `tenantId` entry of MDC in `MeteredCommandProcessor`. And assuming the new logger uses default formatting, and that is includes contents of MDC under `"mdc"` key (which I _think_) happens, I think it should be automatically included.